presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -503,6 +518,21 @@ public static long floorFloat(@SqlType(StandardTypes.REAL) long num)
         return floatToRawIntBits((float) floor(intBitsToFloat((int) num)));
     }
 
+    @Description("inverse of Binomial cdf given numberOfTrials, successProbability parameters and p")
+    @ScalarFunction
+    @SqlType(StandardTypes.INTEGER)
+    public static long inverseBinomialCdf(
+            @SqlType(StandardTypes.INTEGER) long numberOfTrials,
+            @SqlType(StandardTypes.DOUBLE) double successProbability,
+            @SqlType(StandardTypes.DOUBLE) double p)
+    {
+        checkCondition(p >= 0 && p <= 1, INVALID_FUNCTION_ARGUMENT, "p must be in the interval [0, 1]");
```

Comment:
Nits: you are using 0.0 and 1.0 in `binomialCdf` but integer here. Please make them consistent (and consistent with other functions in the class). Thanks!

Reply:
Done